L1Trigger/L1CaloTrigger/python/Phase1L1TJetCalibrator_9x9Jets_cfi.py

```diff
@@ -4,773 +4,162 @@
-            1.0, 1.0, 1.0, 1.0, 1.0, 
-            1.0, 1.0, 1.0, 1.0, 1.0, 
-            1.0, 1.0, 1.0, 1.0, 1.0, 
-            1.0, 1.0, 1.0, 1.0, 1.0, 
-            1.0, 1.0, 1.0, 1.0, 1.0, 
-            1.0, 1.0, 1.0, 1.0, 1.0, 
-            1.0, 1.0, 1.0, 1.0
-        ),
-        l1tPtBins = cms.vdouble(
-            -float('inf'), 26.2286936937, 25.0883854167, 21.5503612231, 19.2925423089, 
-            21.3457232423, 22.8334014207, 23.9319267237, 26.6239039363, 30.3986741142, 
-            33.7250755974, 37.8195972328, 42.1978424806, 52.2604512363, 80.2692022114, 
-            122.67241995, 166.87206295, 199.54597288, 226.44949301, 245.655374, 
-            257.034237337, 257.866298241, 287.3217783, 347.484176974, 307.774717514, 
-            272.791666667, 304.614583333, 366.880208333, 406.911458333, 296.145833333, 
-            209.0, 217.875, 211.25, 178.5, float('inf')
-        )
-    ), 
-    cms.PSet(
-        etaMax = cms.double(4.191),
-        etaMin = cms.double(3.489),
+        etaMin = cms.double(2.172),
         l1tCalibrationFactors = cms.vdouble(
-            1.0, 1.0, 1.0, 1.0, 1.0, 
-            1.0, 1.0, 1.0, 1.0, 1.0, 
-            1.0, 1.0, 1.0, 1.0, 1.0, 
-            1.0, 1.0, 1.0, 1.0, 1.0, 
-            1.0, 1.0, 1.0, 1.0, 1.0, 
-            1.0, 1.0, 1.0
+            1.33772238866, 1.33772238866, 1.33772238866, 1.33473934195, 1.30242829723, 
+            1.27631075498, 1.25143840782, 1.2365993847, 1.22283208594, 1.21383948973, 
+            1.20270942905, 1.19242313522, 1.18157505638, 1.17163939573, 1.16212445691, 
+            1.15064512868, 1.12111242416, 1.10180561512, 1.06494795446, 1.02791008468, 
+            0.979565936906, 0.935460737671, 0.922476716325, 0.870214599247, 0.862692426562, 
+            0.780257657402, 0.726159840141, 0.698509844652
         ),
         l1tPtBins = cms.vdouble(
-            -float('inf'), 22.5440519958, 20.5700369268, 18.940228544, 19.656022303, 
-            21.0931801845, 23.113990869, 26.3227763633, 30.3817638115, 35.043540285, 
-            39.4982579153, 43.4506655731, 48.6407605324, 59.5995676838, 82.7690755527, 
-            114.280107948, 142.242281862, 162.02662341, 183.956841939, 210.824209207, 
-            229.926842684, 227.643518519, 238.016534392, 247.125, 298.946428571, 
-            271.65625, 243.03125, 326.625, float('inf')
+            -float('inf'), 15.2987042682, 20.6666592562, 27.150657531, 34.834542074, 
+            42.9223748303, 51.5696382427, 59.9311353298, 68.6376432274, 76.9323193226, 
+            85.3108305134, 95.1635083363, 104.886459308, 114.448100114, 123.396424629, 
+            133.054919468, 151.922665869, 174.391475487, 200.175498357, 244.035295655, 
+            306.180445151, 373.469537815, 415.021825397, 462.511111111, 506.025, 
+            571.5, 670.875, 730.375, float('inf')
         )
     ), 
     cms.PSet(
         etaMax = cms.double(5.191),
-        etaMin = cms.double(4.191),
+        etaMin = cms.double(3),
```

Comment:
There is now an uncovered runge in eta: [2.5 , 3.0]
Is it done on purpose?

Reply:
@EmyrClement Can you please comment?

Reply:
That eta range does appear to be missing from the calibration, I'm taking a look to see what happened.